app/community/index.html

```diff
@@ -99,7 +99,7 @@ <h2>Videos</h2>
         <div>If you're tired of reading, why not check out some videos?</div>
         <ul>
           <li>Ready to get your hands dirty and revolutionize the modern web?
-            Join Rob Dodson from the Chrome Developer Relations team as he
+            Join Elliott Marquez from the Polymer team and Rob Dodson from the Chrome Developer Relations team as he
             explores the ins and outs of Polymer, in his famous
```

Comment:
in his famous => maybe change this sentence to just "in the Polycasts video series" or something?